torch/distributions/multivariate_normal.py

```diff
@@ -66,6 +66,13 @@ def _batch_mahalanobis(bL, bx):
     return reshaped_M.reshape(bx_batch_shape)
 
 
+def _precision_to_scale_tril(P):
+    # Ref: https://nbviewer.jupyter.org/gist/fehiepsi/5ef8e09e61604f10607380467eb82006#Precision-to-scale_tril
+    Lf = torch.cholesky(torch.flip(P, (-2, -1)))
+    L = torch.inverse(torch.transpose(torch.flip(Lf, (-2, -1)), -2, -1))
```

Comment:
Is there a reason you're using the cubic-time `torch.inverse()` here rather than the quadratic-time `torch.triangular_solve()`?

Reply:
Nope, using `torch.triangular_solve` is an additional improvement. I will address it. Thanks!